elastic/kibana
# [Alerting] Fixed Telemetry task: "Cannot read properties of undefined" due to the missing shard on Kibana start

## Summary

Resolves #120341

Changed plugins start to register alerting and actions telemetry tasks when EventLog  ESContext is ready.
How to test :
1. Start local ES snapshot
2. Start local Kibana
(before)
3. Observe that the warnings are in the logs:
```
2021-12-03T12:09:14.536+01:00][WARN ][plugins.actions.usage] Error executing actions telemetry task: TypeError: Cannot read properties of undefined (reading 'totalExecutions')
[2021-12-03T12:09:14.554+01:00][WARN ][plugins.alerting.usage] Error executing alerting telemetry task: TypeError: Cannot read properties of undefined (reading 'avgDuration')
```
(after)
3. No warnings in the logs.
